marconi-core/src/Marconi/Core/Indexer/LastEventIndexer.hs

```diff
@@ -133,11 +133,14 @@ tickSave indexer = do
 
 instance (MonadIO m, MonadError IndexerError m) => IsIndex m a LastEventIndexer where
   index timedEvent indexer = do
-    let (saveTime, indexer') =
+    let setLastEvent = case timedEvent ^. event of
+          Nothing -> id
+          Just e -> lastEvent ?~ e
```

Comment:
Curious, why was this considered a bug?

Reply:
I was considering whether we should have a flag to enforce one behaviour or the other. The issue here is that the events that carry a block don't carry a tip and thus, they set the last event to `Nothing`.